submodules/myOffice/myOffice.js

```diff
@@ -175,7 +175,7 @@ define(function(require) {
 		myOfficeCheckWalkthrough: function() {
 			var self = this;
 
-			if (!monster.apps.auth.currentAccount.hasOwnProperty('trial_time_left')) {
+			if (!monster.apps.auth.currentAccount.hasOwnProperty('trial_time_left') && !self.appFlags.disableFirstUseWalkthrough) {
```

Comment:
The check for `!self.appFlags.disableFirstUseWalkthrough` is no longer necessary, as it is already checked by `monster.pub('myaccount.hasToShowWalkthrough', ....` below.

```suggestion
			if (!monster.apps.auth.currentAccount.hasOwnProperty('trial_time_left')) {
```